.github/workflows/wheels.yml

```diff
@@ -26,17 +26,20 @@ jobs:
           CIBW_BEFORE_BUILD: python -m pip install cmake
           CIBW_TEST_COMMAND: python -m pytest {package}/gph/python/test
           CIBW_TEST_REQUIRES: pytest hypothesis
+          # Should generate universal2 wheels for CP3.8 and CP3.9
+          CIBW_ARCHS_MACOS: x86_64 universal2
 
       # SciPy do not ship manylinux2010 wheels for Python 3.10, so we too build ours for manylinux2014
       - name: Build wheels for Python 3.10
-        uses: pypa/cibuildwheel@v2.2.2
+        uses: pypa/cibuildwheel@v2.3.1
         env:
           CIBW_BUILD: "cp310-*"
           CIBW_SKIP: "*-win32 *-musllinux_x86_64 *_i686"
           CIBW_BEFORE_BUILD: python -m pip install cmake
           CIBW_TEST_COMMAND: python -m pytest {package}/gph/python/test
           CIBW_TEST_REQUIRES: pytest hypothesis
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+          CIBW_ARCHS_MACOS: x86_64 universal2
```

Comment:
Perhaps this should be `x86_64 arm64 universal2` to solve my issues?

Reply:
I added this, re-generated the wheels, but still get the same error as before!

Could this be related? https://github.com/Unidata/netcdf4-python/issues/1148#issuecomment-1017645753